Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityPowerMockTest.java

```diff
@@ -96,15 +98,51 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
     }
 
     @Test
-    public void onPaymentMethodSelected_startsPayPal() {
+    public void onPaymentMethodSelected_startsPayPalRequestBillingAgreement() {
+        DropInRequest dropInRequest = new DropInRequest();
+        mActivity.setDropInRequest(dropInRequest);
+        mockStatic(PayPal.class);
+        doNothing().when(PayPal.class);
+        PayPal.requestBillingAgreement(any(BraintreeFragment.class), any(PayPalRequest.class));
+
+        mActivity.onPaymentMethodSelected(PaymentMethodType.PAYPAL);
+
+        verifyStatic();
+        PayPal.requestBillingAgreement(eq(mActivity.braintreeFragment), any(PayPalRequest.class));
+    }
+
+    @Test
+    public void onPaymentMethodSelected_startsPayPalRequestOneTimePayment() {
+        PayPalRequest paypalRequest = new PayPalRequest("10")
+                .currencyCode("USD");
+        DropInRequest dropInRequest = new DropInRequest()
+                .paypalRequest(paypalRequest);
+        mActivity.setDropInRequest(dropInRequest);
+        mockStatic(PayPal.class);
+        doNothing().when(PayPal.class);
+        PayPal.requestOneTimePayment(any(BraintreeFragment.class), any(PayPalRequest.class));
+
+        mActivity.onPaymentMethodSelected(PaymentMethodType.PAYPAL);
+
+        verifyStatic();
+        PayPal.requestOneTimePayment(eq(mActivity.braintreeFragment), any(PayPalRequest.class));
+    }
+
+    @Test
+    public void onPaymentMethodSelected_startsPayPalWIthRequestDefaultsToBillingAgreement() {
```

Comment:
I'm a little confused on the title of this test and what it is testing. Is this supposed to test when no `PayPalRequest` is passed in?